WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardReadersViewViewModel.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import SwiftUI
+
+struct Manual: Identifiable {
+    let id: Int
+    let image: UIImage
+    let name: String
+    let urlString: String
+}
+
+final class CardReadersViewViewModel {
```

Comment:
No need for the repeated View here. All ViewModels relate to a View.

Reply:
Thanks for the clarification! It definitely sounded weird when I was trying to name this 😅